packages/web-client/app/components/card-space/create-space-workflow/index.ts

```diff
@@ -234,6 +234,18 @@ export default class CreateSpaceWorkflowComponent extends RestorableWorkflowComp
     return CreateSpaceWorkflow;
   }
 
+  get currentCardSpaceDetails() {
+    return {
+      profilePhoto: this.workflow.session.getValue('profileImageUrl'),
+      coverPhoto: this.workflow.session.getValue('coverPhotoUrl'),
+      name: this.workflow.session.getValue('displayName'),
+      host: this.workflow.session.getValue('url'),
+      category: this.workflow.session.getValue('profileCategory'),
+      description: this.workflow.session.getValue('profileDescription'),
+      buttonText: this.workflow.session.getValue('buttonText'),
```

Comment:
@jurgenwerk I've introduced the workflow session keys 'coverPhotoUrl' and 'profileDescription' here. Realising that our naming of keys for this workflow is not very consistent. Should all keys be prepended with profile?

Reply:
Nice. Yeah would be good for consistency (also to match the naming with the fields from the hub).